Agreements/Internal/Solent_S.txt

```diff
@@ -0,0 +1,3 @@
+COPX:EGHH:*:NEDUL:THRED:*:SOLENT:London AC Wothing:10000:*:^NEDUL
+
+COPX:EGHH:*:*:THRED:*:SOLENT:London AC Worthing:10000:*:^THRED
```

Comment:
EGHI Versions?

Reply:
Resolved this. Line 1 should have been EGHI, line 3 EGHH.